clang/lib/Driver/ToolChains/Cygwin.cpp

```diff
@@ -0,0 +1,109 @@
+//===--- Cygwin.cpp - Cygwin ToolChain Implementations ----------*- C++ -*-===//
```

Comment:
In new code, can remove the comments per the updated https://llvm.org/docs/CodingStandards.html#file-headers

Reply:
Like so?